apps/frontend/src/components/Class/Overview/index.tsx

```diff
@@ -1,24 +1,212 @@
-          <p className={styles.description}>{_class.course.requirements}</p>
-        </>
-      )}
+      <div>
+        {(() => {
+          switch (responses.Attendance) {
+            case Consensus.Yes:
+              return (
+                <>
+                  <User className={styles.icon} />
+                  <span className={styles.userSubmissionDescription}>
+                    Attendance Required
+                  </span>
+                </>
+              );
+            case Consensus.No:
+              return (
+                <>
+                  <UserXmark className={styles.icon} />
+                  <span className={styles.userSubmissionDescription}>
+                    Attendance Not Required
+                  </span>
+                </>
+              );
+          }
+        })()}
+      </div>
+      <div>
+        {(() => {
+          switch (responses.Recording) {
+            case Consensus.Yes:
+              return (
+                <>
+                  <VideoCamera className={styles.icon} />
+                  <span className={styles.userSubmissionDescription}>
+                    Lectures Recorded
+                  </span>
+                </>
+              );
+            case Consensus.No:
+              return (
+                <>
+                  <VideoCameraOff className={styles.icon} />
+                  <span className={styles.userSubmissionDescription}>
+                    Lectures Not Recorded
+                  </span>
+                </>
+              );
+          }
+        })()}
+      </div>
+      <div>
+        {(responses.Attendance === Consensus.Indeterminate ||
+          responses.Attendance === Consensus.BellowThreshold) && (
+          <>
+            <QuestionMark
+              className={styles.icon}
+              style={{ color: "var(--label-color)" }}
+            />
+            <span
+              className={styles.userSubmissionDescription}
+              style={{ color: "var(--label-color)" }}
+            >
+              Unknown Attendance Requirement
+            </span>
+          </>
+        )}
+      </div>
+      <div>
+        {(responses.Recording === Consensus.Indeterminate ||
+          responses.Recording === Consensus.BellowThreshold) && (
+          <>
+            <QuestionMark
+              className={styles.icon}
+              style={{ color: "var(--label-color)" }}
+            />
+            <span
+              className={styles.userSubmissionDescription}
+              style={{ color: "var(--label-color)" }}
+            >
+              Unknown Recording Status
+            </span>
+          </>
+        )}
+      </div>
+      <Link
+        to="ratings?feedbackModal=true"
+        className={styles.suggestEdit}
+        onClick={handleFeedbackClick}
+      >
+        Look inaccurate? Suggest an edit →{" "}
+      </Link>
     </div>
   );
 }
+
+function getResponse(course: ICourse, metricName: MetricName): Consensus {
```

Comment:
If helper functions are used regularly, I'd place them in a common file in `lib` or in your `@repo/shared` package